qiskit/extensions/unitary.py

```diff
@@ -56,12 +56,17 @@ class UnitaryGate(Gate):
             circuit.append(gate, [0, 1])
     """
 
-    def __init__(self, data, label=None):
+    def __init__(self, data, label=None, skip_check=False):
```

Comment:
That's a good option to have 👍🏻 We have a similar option in the SK passes, where it's called `check_input` -- do we want to call it the same here for consistency?

Reply:
Sure, I'll rename the option.